### DISCUSSION TITLE
Close code is None after closing socket on frontend (node.js)

### DISCUSSION THREAD

**Discussion by PetrShchukin** [django/channels#1568] [Q&A]
Description: I have frontend on React, after some events like WebSocket.close() or when pressing 'back' button in a browser I'm getting close_code = None in channels. Shouldn't it be a particular code?
Steps to reproduce:
1) Close web socket on node.js application
2) Look at the close_code inside disconnect method

Expected behavior:
close_code should be a number value (0-4999)

Actual behavior: 
close_code is None 

Version: 3.0.1

**Comment by carltongibson** [comment]
I think it'll depend on your consumer... 

https://github.com/django/channels/blob/a32b5137c1995c572bb6918b78d7a572d1793058/channels/generic/websocket.py#L83-L90

(I think this is better in discussions. It's not really an issue per se.)

**Comment by carltongibson** [comment]
> (I think this is better in discussions. It's not really an issue per se.)

Don't seem to be able to convert just now.
